.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Validate Gradle Wrapper'
+on: [push, pull_request]
```

Comment:
The on `pull_request` will be ineffective, since this only ever runs in the context of a contributor's personal fork. Also, you will want to exclude the master branch and any pr branches, as is done in [test.yml](https://github.com/openjdk/jtreg/blob/master/.github/workflows/test.yml#L3C1-L8C1), to avoid unnecessary test runs.